tsconfig.json

```diff
@@ -7,7 +7,7 @@
   "compilerOptions": {
     "paths": {
       "@aries-framework/rest": ["./src"],
-      "@src": ["src"],
+      "@src": ["./src"],
```

Comment:
IDE was giving me the error 
```
Non-relative paths are not allowed when 'baseUrl' is not set. Did you forget a leading './'
```

Reply:
I haven't encountered this error, but you might try adding to the tsconfig `"BaseUrl": "."`